plugins/woocommerce/changelog/analytics-order-import-action

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Comment: Add action 'woocommerce_order_scheduler_after_import_order' fired after WooCommerce Analytics imports an order or refund and invalidates reports cache.
```

Comment:
```suggestion
Add action 'woocommerce_order_scheduler_after_import_order' fired after WooCommerce Analytics imports an order or refund and invalidates reports cache.
```

Any reason you initially added a changelog entry, but later switched to adding it as a comment instead in  https://github.com/woocommerce/woocommerce/pull/60827/commits/169e78c85f993cd644f312dedca439a9169e4ab7#diff-27fadf9a7d8592f372cd8e145eff6c54c25c4ba19e7f03d85522c7e08f2186ac

I think it would be helpful to add a changelog entry for this change, allowing community developers to be informed through the changelog.